src/slicegpt/data_utils.py

```diff
@@ -59,6 +59,51 @@ def get_dataset(name: str) -> datasets.DatasetDict:
     return ds
 
 
+def prepare_test_dataloader(
+    dataset: datasets.Dataset, tokenizer: PreTrainedTokenizerBase, seqlen: int = 2048, batch_size: int = 1
+) -> DataLoader[dict[str, torch.Tensor]]:
+    """
+    Get a DataLoader from a test dataset. This dataloader is used for evaluation on the entirety of the given dataset.
```

Comment:
I would add a note that this data loader is meant to match exactly the approach used in other papers.

Reply:
Done